azure_monitor/examples/client.py

```diff
@@ -0,0 +1,16 @@
+import requests
+
+from azure_monitor import AzureMonitorSpanExporter
```

Comment:
Do we really need azure_monitor folder?, feels redundant

Reply:
For future exporters to different ingestion endpoints, they can live in separate folders.

Reply:
Now that the repo is called differently I totally agree